cmd/snap/inhibit.go

```diff
@@ -25,12 +25,30 @@ import (
 	"os"
 	"time"
 
+	"github.com/godbus/dbus"
 	"github.com/snapcore/snapd/cmd/snaplock/runinhibit"
 	"github.com/snapcore/snapd/i18n"
 	"github.com/snapcore/snapd/progress"
 	"github.com/snapcore/snapd/usersession/client"
 )
 
+func detectSnapDesktopIntegration(snapName string) bool {
+	if !isGraphicalSession() {
+		return false
+	}
+	// Check if Snapd-Desktop-Integration is available
+	conn, err := dbus.SessionBus()
+	if err == nil {
```

Comment:
Sorry, totally missed this one, I'd reverse this too, then we can remove the last level of indentation
```suggestion
if err != nil {
    // maybe log this?
    // logger.Noticef(...)
    return false
}
```

Reply:
Done.